userguide/dailyuse/anbox.rst

```diff
@@ -97,19 +105,19 @@ Keep your apps up to date
 How to uninstall apps
 ---------------------
 
-- To uninstall apps, run ``adb uninstall [APP_ID]`` from your Ubuntu Touch device::
+- To uninstall apps, run ``adb uninstall [APP_ID]`` from your computer::
```

Comment:
mhm. I see where you are coming from. you are typing on your computer, so you are running this "from your computer". we are running it "on the UT device". How about we simply drop the clause here. It's explained below anyway. Let's just say

To uninstall apps, run the following commands

Or

You can uninstall apps with ``adb uninstall``, see below

Reply:
Very unclear to me. My understanding is the command runs on the computer, and removes apps from the device.